src/index.js

```diff
@@ -641,7 +641,14 @@ try {
         '  | (_| |/ /| |_) | |_) |',
         '   \\__,_/___|_.__/|_.__/ ',
         '',
-        '  A tool for deploying Azure infrastructure based on proven practices.'
+        '  A tool for deploying Azure infrastructure based on proven practices.',
+        '',
+        '  Important: The Azure Building Blocks project will not be undergoing',
+        '  further development.  For advanced deployment template capabilities',
+        '  refer to Bicep(http://aka.ms/bicep), Azure Resource Manager template',
+        '  specs(http://aka.ms/templatespecs) and the Azure Resource Manager',
+        '  template documentation(https://aka.ms/arm-template-docs). New',
```

Comment:
```suggestion
        '  refer to Bicep (https://aka.ms/bicep), Azure Resource Manager template',
        '  specs (https://aka.ms/templatespecs) and the Azure Resource Manager',
        '  template documentation (https://aka.ms/arm-template-docs). New',
```
http -> https and some spaces